src/lib/js/value.rs

```diff
@@ -592,7 +592,15 @@ impl Display for ValueData {
             ValueData::Function(ref v) => match *v.borrow() {
                 Function::NativeFunc(_) => write!(f, "function() {{ [native code] }}"),
                 Function::RegularFunc(ref rf) => {
-                    write!(f, "function({}){}", rf.args.join(", "), rf.expr)
+                    write!(f, "function(")?;
+                    let last_index = rf.args.len() - 1;
+                    for (index, arg) in rf.args.iter().enumerate() {
```

Comment:
This loop can be rewritten using `.iter().join()`.

Reply:
An attempt was made, but I'm having trouble converting `rf.args` into a format that supports `join`.